config/locales/user_mailer/en.yml

```
@@ -118,6 +118,16 @@ en:
       verifying_step_proof_of_address: 'If you try to verify your identity in person
         again, you need to bring a valid proof of address if your current
         address is different than the address on your ID.'
+    in_person_failed_suspected_fraud:
+      body:
+        help_center_html: If you need further help, you can <a
+          href="%{help_center_url}">visit our Help Center</a> or reach out to
+          the agency you are trying to access.
+        intro: We understand that you were attempting to verify your identity through
+          Login.gov, however your identity could not be verified at the
```

```suggestion
          %{app_name}, however your identity could not be verified at the
```
We should always use `app_name` instead of hardcoding `Login.gov`